### DISCUSSION TITLE
[1809540] Nine Sols/九日 {本体}

### DISCUSSION THREAD

**Discussion by Zegasen** [pjy612/SteamManifestCache#722] [Share]
### Steam 游戏ID

1809540

### 密钥列表

```hash
1809541;6c41504f601faddc7ec40aff4688b9ee9e34c37f2ab46b882880cbcc76a8eb40
1809542;39cff11ed94d5fb43612916d75ca9d971b35caeca4fa3bda91610e1118927ec3
```

### 清单附件

[1809540.zip](https://github.com/user-attachments/files/16661872/1809540.zip)

### 备注

code:EA01E8B2BAB53B6C
email:[Zeglisas@outlook.com]

**Comment by sanaousuke** [comment]
please update the manifest...

**Reply by Zegasen** [reply]
更新

**Reply by sanaousuke** [reply]
thnaks, but please update manifest to 6561561052042428226

**Reply by Zegasen** [reply]
ok

**Reply by sanaousuke** [reply]
thanks your great!
